tests/unit_tests.cpp

```diff
@@ -0,0 +1,133 @@
+#include "./lib/catch.hpp"
+#include "choices.hpp"
+#include "feature.hpp"
+#include "model.hpp"
+#include "polymer.hpp"
+#include "reaction.hpp"
+#include "tracker.hpp"
+
+TEST_CASE("Genome construction")
+{
+    auto plasmid = std::shared_ptr<Genome>(new Genome("T7", 305, 0.01, 0.001, 9.0, 20.0));
+    auto plasmid2 = std::shared_ptr<Genome>(new Genome("T7", 305));
+    
+    REQUIRE(plasmid->transcript_degradation_rate() == 0.01);
+    REQUIRE(plasmid->transcript_degradation_rate_ext() == 0.001);
+    REQUIRE(plasmid->rnase_speed() == 9);
+    REQUIRE(plasmid->rnase_footprint() == 20);
+
+    //Test the genome default constructor
+    REQUIRE(plasmid2->transcript_degradation_rate() == 0.0);
+    REQUIRE(plasmid2->transcript_degradation_rate_ext() == 0.0);
+    REQUIRE(plasmid2->rnase_speed() == 0.0);
+    REQUIRE(plasmid2->rnase_footprint() == 0);
+}
+
+TEST_CASE("Add mask to genome")
+{
+    auto plasmid = std::shared_ptr<Genome>(new Genome("T7", 305));
+    std::vector<std::string> interactions = {"rnapol","ecolipol"};
+    plasmid->AddMask(15,interactions);
```

Comment:
Lines 29 and 30: There should be spaces after each comma.

Reply:
Fixed.